src/megachatapi_impl.cpp

```diff
@@ -5241,6 +5278,15 @@ void MegaChatCallHandler::onLocalMediaError(const string errors)
 
 void MegaChatCallHandler::onRingOut(Id peer)
 {
+    assert(chatCall != NULL);
+    if (chatCall != NULL)
+    {
+        if (chatCall->getRemoteStatus() == rtcModule::ICall::kStateInitial)
```

Comment:
Is there any other possible status in this callback? if so, we would miss the notification to the app. If not, we should at least log a message (plus an assertion, maybe).

Reply:
The condition it is necessary to avoid notify several times Ring-In state when there are many clients